### ISSUE TITLE
slic3r configuration

### ISSUE THREAD

**Issue by excogitation** [UP3D-gcode/UP3D#13]
how about adding a working slic3r configuration file to ease getting started?

**Comment by Jonchua** [comment]
I have difficulty with slic3r also. The print head is at a large distance from the print platform surface even after adjusting the nozzle height parameter.

**Comment by mvadu** [comment]
I did create one for my Cetus. Not sure if its same for other TierTime printers. https://www.thingiverse.com/thing:3045287

**Comment by maslovw** [comment]
@mvadu, your configuration is for UPStudio. This project is a bit different, here people tried to reverse-engineer tiertime printers and make GCode -> upcode interpreter. It was started before UPStudio had GCode support

**Comment by Ljun-1994** [comment]
> I did create one for my Cetus. Not sure if its same for other TierTime printers. https://www.thingiverse.com/thing:3045287

Hello, may I ask if you need to modify the firmware for Prusa slicing?

**Comment by mvadu** [comment]
> > I did create one for my Cetus. Not sure if its same for other TierTime printers. https://www.thingiverse.com/thing:3045287
> 
> Hello, may I ask if you need to modify the firmware for Prusa slicing?

No, this was with default tier time firmware.. It's just config you import into sli3r (there was no Prusa Slicer when I did this) and generate gcode. Which you then import into up studio and print..

**Comment by Ljun-1994** [comment]
> > > I did create one for my Cetus. Not sure if its same for other TierTime printers. https://www.thingiverse.com/thing:3045287
> > 
> > 
> > Hello, may I ask if you need to modify the firmware for Prusa slicing?
> 
> No, this was with default tier time firmware.. It's just config you import into sli3r (there was no Prusa Slicer when I did this) and generate gcode. Which you then import into up studio and print..

![image](https://github.com/UP3D-gcode/UP3D/assets/136558671/352e5750-70cb-4506-bcea-f5cb9740edf4)
![image](https://github.com/UP3D-gcode/UP3D/assets/136558671/a162c743-cc13-404c-84ba-26fc63822d99)
My UP BOX+and UP Plus 2

Is it okay to set the image like this?
